client/src/docs-ui/landing-hero-cta/landing-hero-cta.tsx

```diff
@@ -41,7 +41,7 @@ export class DocsLandingHeroCTA {
             />
           </docs-internal-link>
           <docs-internal-link
-            href="/start/"
+            href="/start"
```

Comment:
Shouldnt there be a trailing forward slash for this?

Reply:
No, the `/start` page exists whereas the `/start/` page doesn't.  While one is redirected to the other, `/start` is correct here.  We could theoretically remedy this instead by making trailing slash pages correct for all URLs, but it seems better to have this conform to the standard already in place.